queue_job/jobrunner/runner.py

```diff
@@ -432,6 +432,7 @@ def process_notifications(self):
                     break
                 notification = db.conn.notifies.pop()
                 uuid = notification.payload
+                _logger.debug("received notification on db %s: %s", db.db_name, uuid)
```

Comment:
Is that not too noisy?

Reply:
it is indeed very noisy, and can even be confusing and lead to think that there is an issue while there is none, removed